packages/muelu/test/scaling/CMakeLists.txt

```diff
@@ -289,6 +289,14 @@ IF (${PACKAGE_NAME}_HAVE_TPETRA_SOLVER_STACK)
     COMM mpi # HAVE_MPI required
   )
 
+  TRIBITS_ADD_TEST(
+    Driver
+    NAME "SchwarzSmoothingWithAverages"
+    ARGS "--linAlgebra=Tpetra --xml=oLapSchwarzEquivBlkSmoo.xml --belosType=\"Fixed\ Point\" --rowmap=GblkMap.dat --matrix=GblkAmat.dat --rhs=GblkRhs.dat --tol=.1"
```

Comment:
Does this need the `regularOverlap.dat` file anywhere? I see it referenced in comments, and I see it's copied to the build dir, but I don't see it explicitly called by anything.

Reply:
first I thought I added the wrong test to CMakefile so then I pushed the "other test" ... now I realize that I actually had the test I wanted. Anyway, I just pushed a second change to the PR that now does both tests: averaging with block relaxation and averaging with schwarz. the block relaxation one uses the regularOverlap.dat file to do user defined blocks that just happen to correspond to the overlapping schwarz blocks.  Thanks for catching this potentially confusing issue.

Reply:
See @rstumin 's push